tests/gocase/unit/type/tdigest/tdigest_test.go

```diff
@@ -587,4 +587,73 @@ func tdigestTests(t *testing.T, configs util.KvrocksServerConfigs) {
 			require.EqualValues(t, rank, expected[i])
 		}
 	})
+
```

Comment:
Maybe we could add a case with more different or same and unordered inputs to have a comprehensive integration test.